string-manipulation-3/src/is-palindromic.js

```diff
@@ -0,0 +1,15 @@
+/* exported isPalindromic */
+function isPalindromic(string) {
+  var reverseString = '';
+  var stringArray = string.split(' ');
+  var stringNoSpaces = '';
+  for (var i = 0; i < string.length; i++) {
+    if (string[string.length - 1 - i] !== ' ') {
+      reverseString = reverseString + string[string.length - 1 - i];
+    }
+  }
+  for (i = 0; i < stringArray.length; i++) {
+    stringNoSpaces = stringNoSpaces + stringArray[i];
+  }
```

Comment:
Since the time you wrote this solution, you might have some ideas on how to do this reversing and comparison in a less brute force way. If not, let me know, and I can offer some suggestions.